framework/src/outputs/png/PNGOutput.C

```diff
@@ -25,8 +26,12 @@ validParams<PNGOutput>()
   params.addParam<bool>("transparent_background",
                         false,
                         "Determination of whether the background will be transparent.");
-  params.addParam<unsigned int>("resolution", 1, "The resolution of the image.");
-  MooseEnum color("GRAY BRYW BCR RWB BR");
+  params.addParam<unsigned int>("resolution", 25, "The resolution of the image.");
```

Comment:
addRangeCheckedParam here - make sure the resolution is non-negative. The zero case results in an FPE in your logic below. Negative is also bad.